app/helpers/tasks_helper.rb

```diff
@@ -0,0 +1,13 @@
+module TasksHelper
+  def display_task_completed_at(completed_at)
+    return completed_at ? "Completed: #{completed_at.to_formatted_s(:twelvehr_long_ordinal)}" : "In Progress"
+  end
+
+  def complete_button_text(completed_at)
+    return completed_at ? "Mark In-Progress" : "Mark Complete"
+  end
```

Comment:
Nice us of the ternary operator and helper methods to make for great user experience.